near-sdk-sim/README.md

```diff
@@ -26,79 +34,430 @@ near-sdk = { git = "https://github.com/near/near-sdk-rs.git", tag="3.0.0" }
 
 ```
 
+Note that you need to add the `tag` (or `commit`) of the version.
+
+
+## Workspace setup
+
+If you want to check gas & storage usage of one Rust contract, you can add the above dependencies to `Cargo.toml` in the root of your project. If you want to test cross-contract calls, we recommend setting up a cargo [workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html). Here's how it works:
+
+Let's say you have an existing contract project in a folder called `contract`.
+
+Go ahead and make a subfolder within it called `contract`, and move the original contents of `contract` into this subfolder. Now you'll have `contract/contract`. You can rename the root folder to something like `contracts` or `contract-wrap`, if you want. Some bash commands to do this:
+
+```bash
+mkdir contract-wrap
+mv contract contract-wrap
+```
+
+Now in the root of the project (`contract-wrap`), create a new `Cargo.toml`. You'll have to add the normal `[package]` section, but unlike most projects you won't have any `dependencies`, only `dev-dependencies` and a `workspace`:
+
+
+```toml
+[dev-dependencies]
+near-sdk = { git = "https://github.com/near/near-sdk-rs.git", tag="3.0.0" }
+near-sdk-sim = { git = "https://github.com/near/near-sdk-rs.git", tag="3.0.0" }
```

Comment:
What's the plan to update it? Pre-publishing?

Reply:
I spoke with @willemneal and he thinks it makes sense to publish the package with this, then make a small doc update right after.